homeassistant/components/xiaomi_miio/sensor.py

```diff
@@ -250,3 +268,67 @@ def device_class(self):
     def state(self):
         """Return the state of the sensor."""
         return self._sub_device.status[self._data_key]
+
+
+class XiaomiGatewayIlluminanceSensor(Entity):
+    """Representation of the gateway device's illuminance sensor."""
+
+    def __init__(self, gateway_device, gateway_name, gateway_device_id):
+        """Initialize the entity."""
+        self._gateway = gateway_device
+        self._name = f"{gateway_name} Illuminance"
+        self._gateway_device_id = gateway_device_id
+        self._unique_id = gateway_device_id
```

Comment:
If we add more sensor types in the future, it's good to add the type to this sensor unique id now.

Reply:
I reverted that change